README.md

```diff
@@ -45,26 +45,8 @@ $ docker service create \
   dockersamples/visualizer
 ```
 
-## Running on ARM
```

Comment:
You should not remove this section. Someone will still have to build and maintain an image on the Docker Hub.